### DISCUSSION TITLE
Looking for a new maintainer

### DISCUSSION THREAD

**Discussion by llucax** [mateusfccp/pausable_timer#55] [General]
It's been almost 2 years since the last time I used Dart or Flutter, and while I enjoy maintaining open source projects, it is becoming more and more difficult to keep up to date with changes in Dart and maintaining this project, so **I'm looking for a maintainer willing to take over this project** to ensure its health in the future.

Please leave a message here if you are interested.

**Comment by mateusfccp** [comment]
Hey, @llucax, you may transfer the repository/package to me if you want!

I currently work actively with Flutter and Dart, so I can spare some time to take care of this project.

Thanks!

**Reply by llucax** [reply]
Hi @mateusfccp, thanks for the offer! I just sent you an e-mail to the e-mail published in your GitHub profile, I hope that's OK!

**Comment by llucax** [comment]
Thanks @mateusfccp for stepping up and shipping a new release! :heart: :tada: 

If for some chance any user of this library is reading this, I encourage you to say hi and thanks to @mateusfccp :wave:
